perceval/backends/github.py

```diff
@@ -68,6 +69,34 @@ def __init__(self, owner=None, repository=None, token=None,
         self.owner = owner
         self.repository = repository
         self.client = GitHubClient(owner, repository, token, base_url)
+        self._users = {}  # internal users cache
+
+    def _get_user(self, issue, field):
+        """ Add the users data to the item """
+
+        user = {}
+
+        fields = ['user', 'assignee']
+
+        if field not in fields:
+            logging.warning("Wrong user field %s" % (field))
+            return {}
+
```

Comment:
You don't need to include all this code if you just pass to the method the `login` value. Neither `issue` nor `field` are needed here to get the user information.

For instance:
```python
def _get_user(self, login):
    if not login:
        return {}

    user_raw = self.client.get_user(login)
    user_orgs_raw = self.client.get_user_orgs(login)
    [...]